cockatrice/src/settings/shortcuts_settings.h

```diff
@@ -590,11 +590,23 @@ class ShortcutsSettings : public QObject
         {"DlgLoadDeckFromClipboard/refreshButton", ShortcutKey(QT_TRANSLATE_NOOP("shortcutsTab", "Refresh"),
                                                                parseSequenceString("F5"),
                                                                ShortcutGroup::Load_deck)},
+        {"Replays/aSkipForward", ShortcutKey(QT_TRANSLATE_NOOP("shortcutsTab", "Skip Forward"),
+                                             parseSequenceString("Right"),
+                                             ShortcutGroup::Replays)},
+        {"Replays/aSkipBackward", ShortcutKey(QT_TRANSLATE_NOOP("shortcutsTab", "Skip Backward"),
+                                              parseSequenceString("Left"),
+                                              ShortcutGroup::Replays)},
+        {"Replays/aSkipForwardBig", ShortcutKey(QT_TRANSLATE_NOOP("shortcutsTab", "Skip Forward by a lot"),
+                                                parseSequenceString("Ctrl+Right"),
+                                                ShortcutGroup::Replays)},
+        {"Replays/aSkipBackwardBig", ShortcutKey(QT_TRANSLATE_NOOP("shortcutsTab", "Skip Backward by a lot"),
+                                                 parseSequenceString("Ctrl+Left"),
+                                                 ShortcutGroup::Replays)},
         {"Replays/playButton", ShortcutKey(QT_TRANSLATE_NOOP("shortcutsTab", "Play/Pause"),
                                            parseSequenceString("Space"),
                                            ShortcutGroup::Replays)},
         {"Replays/fastForwardButton", ShortcutKey(QT_TRANSLATE_NOOP("shortcutsTab", "Toggle Fast Forward"),
-                                                  parseSequenceString("Ctrl+Right"),
+                                                  parseSequenceString("."),
```

Comment:
I'm not sure I agree with bounding something to a non-control key like this. @ebbit1q what do you think?

Reply:
I think . is a bit of a strange choice, maybe just set it to ctrl f, for fast forward

Reply:
gah, just realized that `Ctrl+F` is already used for fullscreen

Reply:
yeah, there is no check for duplicates in the default shortcuts